### DISCUSSION TITLE
Weird "night mode" on the Nord theme?

### DISCUSSION THREAD

**Discussion by popovidis** [saadeghi/daisyui#3235] [Q&A]
Hi there, I am using Nextjs14 + DaisyUI with the Nord theme and during "night hours" my website looks terrible, I am not sure what is triggering this because my friend in another timezone sent me a picture of it being the case and I have been able to replicate it on my phone ~7 am. I believe this is something to do with the Brave browser but I am not entirely sure since I am not sure what is triggering it. I will try another browser during night hours and report back. If someone knows what I can do to stop this from triggering pls let me know. Thank you.

Normal:
![Screenshot from 2024-10-16 10-20-44](https://github.com/user-attachments/assets/d2bc6829-d104-4e17-8c5e-217f90274928)

Weird night mode:
![photo_2024-10-16_10-20-33](https://github.com/user-attachments/assets/46cec1da-a056-40ab-b8ad-f546b7d3b290)

**Comment by saadeghi** [comment]
Please share the link. I can't debug a screenshot.

**Reply by popovidis** [reply]
https://revnu.me/

**Reply by popovidis** [reply]
Just had it happen again on another device with Brave

**Comment by saadeghi** [comment]
There's a style in your CSS, which is not from daisyUI. Did you add it?
```
:root {
  --foreground-rgb: 0,0,0;
  --background-start-rgb: 214,219,220;
  --background-end-rgb: 255,255,255
}

@media (prefers-color-scheme: dark) {
  :root {
    --foreground-rgb:255,255,255;
    --background-start-rgb: 0,0,0;
    --background-end-rgb: 0,0,0
  }
}

body {
  color: rgb(var(--foreground-rgb));
  background: linear-gradient(to bottom,transparent,rgb(var(--background-end-rgb))) rgb(var(--background-start-rgb))
}
```

**Reply by popovidis** [reply]
Oh my bad... That's there from creating nextjs project. Forgot to delete it 😅. 

I'll test that once at home. Thank you
